libwasmvm/src/cache.rs

```diff
@@ -379,7 +379,7 @@ mod tests {
         assert!(cache_ptr.is_null());
         assert_eq!(error_msg.is_some(), true);
         let msg = String::from_utf8(error_msg.consume().unwrap()).unwrap();
-        assert_eq!(msg, "Error calling the VM: Cache error: Error creating Wasm dir for cache: data provided contains a nul byte");
+        assert_eq!(msg, "Error calling the VM: Cache error: Error creating directory borken\u{0}dir/state: data provided contains a nul byte");
```

Comment:
Could you fix my typo while you're here? borken -> broken. Thanks

Reply:
You know, I thought `borken` was intentional. You wrote an intentionally broken string and then even broke the word "broken" :D